src/main/java/it/chalmers/eventItCal/db/entity/Event.java

```diff
@@ -12,6 +12,65 @@
 @Entity
 @Table(name = "event")
 public class Event {
+
+    public static class Builder {
+        private String name;
+        private Date startTime;
+        private Date endTime;
+        private String description;
+        private String organizer;
+        private String contact;
+        private String[] relevantGroups;
+
+        public Builder setName(String name) {
```

Comment:
`name` instead of `setName`, `startTime` instead `setStartTime`  is standard usually